newsletters/2022-07.md

```diff
@@ -0,0 +1,62 @@
+---
+tags: obf
+---
+
+_Having trouble viewing this email or looking for a permalink? [View this newsletter in your browser](https://github.com/OBF/newsletter/blob/master/newsletters/2022-07.md)._
+
+_This email was sent to the [bosc-announce](https://groups.google.com/g/bosc-announce) and [open-bio-l](http://mailman.open-bio.org/mailman/listinfo/open-bio-l/) mailing lists._
+
+# OBF Newsletter July/August 2022
```

Comment:
Maybe July/August/September now?

Or just say September and change the filename to match.

Reply:
```suggestion
# OBF Newsletter July/August/September 2022
```

Reply:
makes sense